src/components/Footer/index.tsx

```diff
@@ -1,32 +1,123 @@
-import styles from "./styles.module.css";
-import image from "./img/LogoSeminario.jpeg";
-import Image from "next/image";
-
+import styles from "./styles.module.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook } from "@fortawesome/free-brands-svg-icons";
+import { faRss } from "@fortawesome/free-solid-svg-icons";
+import { faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { resourcesData, socialIconsData } from "../../utils/data";
 const Footer = () => {
   return (
     <footer className={styles.footer}>
-      <div className={styles.footerSection}>
-        <div className={styles.footerText}>
-          <p>Sitemap</p>
-          <p>Home</p>
-          <p>Post</p>
-          <p>About</p>
-        </div>
-        <div className={styles.footerText}>
-          <p>Conectate con nosotros</p>
-          <p>Facebook</p>
-          <p>Twitter</p>
-          <p>Tiktok</p>
-        </div>
-        <div className={styles.footerText2}>
-          <p>NEXT<span>js</span></p>
+      <div className={styles.container}>
+        <div className={styles.imageContainer}>
+          <img src="/img/logo.jpeg" alt="Logo del sitio" />
         </div>
-        <div className={styles.footerText2}>
-          <p> ▲Vercel</p>
+
+        <div className={styles.sections}>
+          {resourcesData.map((column, columnIndex) => (
```

Comment:
I have reviewed the code and I found a big concern: we are putting too much code in one file. This is not a good practice. We need to break this code into smaller, reusable components. Remember the Lego pieces analogy we talked about: each piece should be independent and easy to use in different situations.

Please, take time to refactor this code. Make smaller components that can be easily reused and implemented in other parts of the project. This is not just a good practice in development, especially in Next.js, but it's also key to keep our code organized and maintainable in the long run.